packages/bpk-component-split-input/src/BpkInputField.tsx

```diff
@@ -27,47 +25,46 @@ import STYLES from './BpkInputField.module.scss';
 
 const getClassName = cssModules(STYLES);
 
-class BpkInputField extends Component {
-  componentDidUpdate(prevProps) {
+interface Props {
+  id: string;
+  label: string;
+  value?: string;
+  focus: boolean;
+  index: number;
+  name?: string;
+  [key: string]: any;
+}
+
+class BpkInputField extends Component<Props> {
+  componentDidUpdate(prevProps: Props) {
     const { focus } = this.props;
     if (prevProps.focus !== focus && this.input && focus) {
       this.input.focus();
       this.input.select();
     }
   }
 
+  private input: HTMLInputElement | null = null;
+
   render() {
     const { focus, id, index, label, name, value, ...rest } = this.props;
     return (
       <div key={index} className={getClassName('bpk-input-field')}>
         <BpkInput
           id={id}
           autoComplete="off"
-          maxLength="1"
+          name={name || ''}
```

Comment:
With the optional props suggestion above we can then just set this as `name={name}` as we will provide the fallback in the props declaration